Editor/CoreAPI/GameLiftFleetManager.cs

```diff
@@ -37,23 +37,26 @@ public async Task<CreateAnywhereFleetResponse> CreateFleet(string fleetName)
         {
             if (_amazonGameLiftWrapper != null)
```

Comment:
See comment above about "if-guard" pattern. We can remove three layers of nesting from this snippet with this pattern

Reply:
Fixed for here as well.

Reply:
I moved the nominal condition to be last with a separate commit